docs/sdkx-react-native/users.mdx

```diff
@@ -0,0 +1,276 @@
+    <td>
+      <ul>
+        <li>Email format should be valid (should be in the format "my@email.com". Invalid emails will result in the creation of an anonymous user.</li>
+        <li>Leading / trailing spaces are not allowed. Emails with leading / trailing spaces will result in the creation of an anonymous user.</li>
+        <li>Email is always case insensitive (eg. 'MY@EMAIL.COM' is same as 'my@email.com').</li></ul></td>
+  </tr>
+  <tr>
+    <td>userAuthToken</td>
+    <td>Optional</td>
+    <td>A user authentication token generated via Hash-Based Message Authentication Code (HMAC) using SHA256. You should provide an HMAC Digest if you want to ensure that 3rd parties cannot file issues on behalf of your users or update their properties. Details <a href="https://support.helpshift.com/kb/article/what-is-user-identity-verification-and-how-do-i-set-it-up/">here</a>.</td>
+    <td>
+      <ul>
+        <li>A valid HMAC Digest should be provided to ensure your users can file issues. Details <a href="https://support.helpshift.com/kb/article/how-do-i-test-that-i-have-user-identity-verification-set-up-correctly/">here</a>.</li>
+      </ul>
+</td>
+
+  </tr>
+</table>
+
+To create a logged-in user in Helpshift, the use of either `userId` or `userEmail` is required.
+
+Example :
+
+```
+login({
+  userName: 'John Doe',
+  userEmail: 'john.doe@app.co',
+  userId: 'unique-user-id-746501' 
+  });
+
+```
+### Logging out users {#logging-out-users}
+
+#### Overview
+
+Once a user logs out from your app, you should call the Helpshift SDK's `logout()` API to ensure no one else can view this user's conversations.
+
+Example :
+```
+logout()
+```
+<Admonition type="info" title="Note">
+
+The Logout API is expected to be used in conjunction with the Login API.
+
+</Admonition>
+
+### Anonymous users {#anonymous-users}
+
+#### Overview
+
+An anonymous user is someone who can access the app without a userId or userEmail. If a user identifier (user ID and/or email) is not passed via the `Login` API, then Helpshift assumes that the user is anonymous and is not currently logged in.
+
+If your use case involves multiple logged-in/anonymous users using the same device and discussing info during support conversations (which you ideally wouldn't want to be shared across logins), you should use the `clearAnonymousUser(boolean clearAnonymousUser)` API.
+
+1. If clearAnonymousUser is true – anonymous users are cleared whenever any user logs in. Once cleared, such users (and their conversations) cannot be fetched again.
+
+2. If clearAnonymousUser is false – anonymous user data will be saved and if a logged-in user is logged out, you will see the anonymous user’s conversation history.
+
+Example : 
+```
+clearAnonymousUser(true)
+```
+<Admonition type="info" title="Note">
+
+- The ClearAnonymousUserOnLogin API does not impact the logged-in user's experience at all
+- The ClearAnonymousUserOnLogin API will only work if called from logged-in mode.
```

Comment:
This is not true. Where did you get this information from?

Reply:
This is mentioned in https://developers.helpshift.com/sdkx_ios/users/#anonymous-users  and unreal doc as well.

Reply:
Updated. removed `The ClearAnonymousUserOnLogin API will only work if called from logged-in mode.`